tests/patterns/python/standalone_decorator.py

```diff
@@ -0,0 +1,16 @@
+#ERROR:
+@decorator
+def fun():
+  return 42
+
+#ERROR:
+@decorator(1,2,3)
+def grumble():
+  return "wahoo"
+
+#ERROR:
+@decorator("oh")
+@decorator("my")
```

Comment:
Hm, I think I'd expect line 13 to match as well as a user. I believe my proposed change to `G.Any (G.At (...))` will do precisely that because it'll try to match both `G.At (...)` nodes, instead of just capturing the list.

Because we ran into this, can you include another test case with `@$NAME($X)`? Let's also include for each set of tests at least one example that shouldn't match.

Reply:
I've added additional tests to catch what you've described, and indeed it does miss those findings, which is troublesome! I've extended the old tests in `tests/patterns/python/standalone_decorator.{py,sgrep}`, and added a new set of tests for `@$NAME($X)` in `tests/patterns/python/standalone_decorator-named.{py,sgrep}`

Reply:
Also: now we do parse lone decorators in `semgrep` patterns as `any`!